rush.json

```diff
@@ -427,6 +427,13 @@
       "shouldPublish": true
     },
 
+    {
+      "packageName": "@kadena/pactjs-utils",
+      "projectFolder": "packages/libs/pactjs-utils",
+      "tags": ["libs"],
+      "shouldPublish": false
+    },
+
```

Comment:
Minuity: sorting in this file is done within the categories by `packageName`. So `@kadena/pactjs-utils` should be below `@kadena/integration-tests`